docker/build_windows.sh

```diff
@@ -0,0 +1,4 @@
+~/env/python-3.5-conda/Scripts/python.exe -m pip wheel JPype1-0.7.4.tar.gz -w wheelhouse
```

Comment:
Do we need to change this script for every new release? Eventually we can make it version agnostic

Reply:
I am hoping to get all the release scripts moved to an azure pipeline release pipeline.   That would also allow me to build the missing OSX builds.   For now I have just checked in my build and test wheel scripts so that when covid finally arrives there is some record of what I did (and given that i have burned lungs from an industrial accident I am not looking forward to it).